cmd/opampsupervisor/supervisor/commander/commander.go

```diff
@@ -95,6 +96,19 @@ func (c *Commander) Restart(ctx context.Context) error {
 	return c.Start(ctx)
 }
 
+func (c *Commander) ReloadConfigFile() error {
+	if c.cmd == nil || c.cmd.Process == nil {
+		return errors.New("agent process is not running")
+	}
+
+	c.logger.Debug("Sending SIGHUP to agent process to reload config", zap.Int("pid", c.cmd.Process.Pid))
```

Comment:
`SIGHUP` isn't supported on Windows. Did this change break `[e2e-tests-windows / supervisor-test](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/16030957451/job/45232053818?pr=41032)`?
`{"error": "failed to send SIGHUP to agent process: not supported by windows"}`

Reply:
The PR was merged despite the error. Should changes in this module trigger [e2e-tests-windows / supervisor-test](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/16030957451/job/45232053818?pr=41032)?

Reply:
[This element](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/40522/files#diff-239707dd27c36360814b762f18855fdb907bca784df484a72e90d6106daf3de6R231-R234) can be made conditional to skip `SIGHUP` tests on Windows.

Reply:
Will take care of this tomorrow as early as possible. 👍

Reply:
@nenadnoveljic I think your link doesn't work well. Maybe it requires certain files to be expanded for the anchor to work? Could you share your suggestion in a different way?

Reply:
Fixing the tests [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/41070)